src/js/components/stock-movement/inbound/StockMovementInboundList.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { fetchTranslations } from 'actions';
+import StockMovementInboundFilters from 'components/stock-movement/inbound/StockMovementInboundFilters';
+import StockMovementInboundHeader from 'components/stock-movement/inbound/StockMovementInboundHeader';
+import StockMovementInboundTable from 'components/stock-movement/inbound/StockMovementInboundTable';
+
+const StockMovementInboundList = (props) => {
+  const [filterParams, setFilterParams] = useState({
+    destination: {
+      id: props.currentLocation.id,
+      value: props.currentLocation.id,
+      name: props.currentLocation.name,
+      label: props.currentLocation.name,
+    },
+  });
```

Comment:
As commented above - `filterParams` should stay `{}`, but `defaultValues` passed to `FilterForm` should be this.